scripts/run_java_interop_tests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# =============================================================
+#
+#     This is ALMOST same as scripts/ru_interop_tests.sh
+#     Please maintain this accordingly; i.e., if you make a 
+#     change to that file or this one, duplicate the change to
+#     the other file.
+#
+# =============================================================
+
+set -ex
+
+SCRIPT_DIR=$(dirname "$0")
+REPO_DIR=$(realpath "${SCRIPT_DIR}/..")
+
+# Set up
+cd "${REPO_DIR}"
+
+
+# These programs need to be already installed
+progs=(docker docker-compose npm)
+for p in "${progs[@]}"
+do
+  command -v "$p" > /dev/null 2>&1 || \
+    { echo >&2 "$p is required but not installed. Aborting."; exit 1; }
+done
+
+# Build & Start the container with the grpc-web java service and proxy
+docker-compose build java-interop-server
+pid=$(docker run -d --network=host grpcweb/java-interop-server)
+
+# Run interop tests
+docker run --network=host --rm grpcweb/prereqs /bin/bash /github/grpc-web/scripts/docker-run-interop-tests.sh
+
+docker rm -f "$pid"
+
```

Comment:
Can you put L42 - L49 above in here: https://github.com/grpc/grpc-web/blob/master/scripts/run_interop_tests.sh#L49

Then you don't need this `run_java_interop_tests.sh` script.

You can compile your L43 above with https://github.com/grpc/grpc-web/blob/master/scripts/run_interop_tests.sh#L34

Reply:
good idea! 
done.
PTAL